scripts/exec.mjs

```diff
@@ -44,12 +44,10 @@ export function execute(command, args = []) {
     proc.on('exit', (code) =>
       code === 0
         ? resolve(trimNewline(dataBuffer.toString('utf8')))
-        : reject(
-            JSON.stringify({
-              code,
-              error: trimNewline(errorBuffer.toString('utf8')),
-            })
-          )
```

Comment:
I recall adding the stringify to ensure that the error would show up in the console when I first made this function. However, since then, I made this function log errors, so there's no purpose to using `JSON.stringify` here.